components/core/src/os/process/linux.rs

```diff
@@ -12,19 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use libc;
 use std::ffi::OsString;
 use std::path::PathBuf;
 use std::ops::Neg;
 use std::os::unix::process::CommandExt;
 use std::process::{self, Command};
-use time::{Duration, SteadyTime};
 
-use error::{Error, Result};
+use libc::{self, pid_t};
+use time::{Duration, SteadyTime};
 
 use super::{HabExitStatus, ExitStatusExt, OsSignal, ShutdownMethod, Signal};
+use error::{Error, Result};
 
-pub type Pid = libc::pid_t;
```

Comment:
I answered this above here: https://github.com/habitat-sh/habitat/pull/2605#discussion_r126523296

I'm not 10/10 either way on this one, whatever you guys think